Header header
int32 aruco_id
float64 range
